macros/get_time_slice.sql

```diff
@@ -16,16 +16,16 @@
             - Postgres
             - Snowflake
             - BigQuery
-    #}
-  {%- if target.type == 'postgres' -%}
-    CASE 
-      WHEN '{{ start_or_end }}' = 'END'
-      THEN 'epoch'::timestamp + '{{ slice_length }} {{ date_or_time_part }}'::interval * (EXTRACT(epoch FROM ({{ date_or_time_expr }} + '{{ slice_length }} {{ date_or_time_part }}'::interval))::int4 / EXTRACT(epoch FROM '{{ slice_length }} {{ date_or_time_part }}'::interval)::int4)
-      ELSE 'epoch'::timestamp + '{{ slice_length }} {{ date_or_time_part }}'::interval * (EXTRACT(epoch FROM {{ date_or_time_expr }})::int4 / EXTRACT(epoch FROM '{{ slice_length }} {{ date_or_time_part }}'::interval)::int4)
+    */#}
+{%- if target.type == 'postgres' -%}
+    CASE
+        WHEN '{{ start_or_end }}' = 'END'
+            THEN 'epoch' ::timestamp + '{{ slice_length }} {{ date_or_time_part }}' ::interval * (EXTRACT(epoch FROM ({{ date_or_time_expr }} + '{{ slice_length }} {{ date_or_time_part }}' ::interval))::int4 / EXTRACT(epoch FROM '{{ slice_length }} {{ date_or_time_part }}' ::interval)::int4)
```

Comment:
Is sqlfluff not expecting up the `interval`, `timestamp`, and `int4` to be uppercase here?

Reply:
I didn't get any errors about that.

Reply:
That's weird. There were a couple of other places with similar missing behavior (like spaces after commas not being flagged). We can let it go for now instead of continuing to fight `sqlfluff` on it, and come back to it at a later time.